renderdoc/driver/vulkan/vk_pixelhistory.cpp

```diff
@@ -341,6 +470,62 @@ struct VulkanPixelHistoryCallback : public VulkanDrawcallCallback
     return pipe;
   }
 
+  VkShaderModule CreateShaderReplacement(const VulkanCreationInfo::Pipeline::Shader &shader)
+  {
+    const VulkanCreationInfo::ShaderModule &moduleInfo =
+        m_pDriver->GetRenderState().m_CreationInfo->m_ShaderModule[shader.module];
+    auto shaderKey = make_rdcpair<ResourceId, rdcstr>(shader.module, shader.entryPoint);
```

Comment:
It seems picky but I would prefer not to use `auto` for the pair here. Since you're constructing it directly you could do:

```
rdcpair<ResourceId, rdcstr> shaderKey(shader.module, shader.entryPoint);
```

instead.